### PR TITLE
fix date thing

### PR DESCRIPTION
**Pull request by rianfowler**
## Testing done
verified locally 

## Screenshots
screen shot 2019-02-05 at 13 28 59

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx [hunk 1]
```diff
@@ -45,9 +45,7 @@ class SaveInProgressIntro extends React.Component {
         const isExpired = expiresAt.isBefore();
 
         if (!isExpired) {
-          const lastSavedDateTime = moment
-            .unix(savedAt)
-            .format('M/D/YYYY [at] h:mm a');
+          const lastSavedDateTime = savedAt.format('M/D/YYYY [at] h:mm a');
```
Comment (rianfowler) [inline src/platform/forms/save-in-progress/SaveInProgressIntro.jsx]: `savedAt` was turned into a moment type on line 40 so it's not necessary to convert it to `unix` type before formatting it.

**Review by jbalboni — APPROVED** [review]
Looks like this has been broken for like 4 months

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx (hunk 1 above)
Comment (erikphansen) [reply src/platform/forms/save-in-progress/SaveInProgressIntro.jsx]: based on the history of this file, it looks like this has been a problem for at least four months? 😱

**Review by erikphansen — APPROVED** [review]

**Comment by rianfowler** [comment]
I will add a unit test for it. but I'm going to try to get this out in a release first real quick.

**Pull request merged**